ui/src/app/styles/colors.ts

```diff
@@ -53,6 +53,11 @@ export default {
     stopped: '#F8C954',
     error: '#DB3214',
   },
+  notificationBanner: {
+    expiring: `rgb(196, 222, 244)`,
```

Comment:
We should be really explicit about these to avoid confusion.  How about:
`expiring_after_two_weeks, expiring_within_two_weeks, expiring_within_one_week`